images/macos/helpers/Common.Helpers.psm1

```diff
@@ -28,16 +28,22 @@ function Get-EnvironmentVariable($variable) {
 function Get-OSVersion {
     $osVersion = [Environment]::OSVersion
     $osVersionMajorMinor = $osVersion.Version.ToString(2)
+    # Monterey needs future review:
+    # [Environment]::OSVersion returns 11.0 for Monterey preview.
+    $osMontereyVersion = [SemVer]$(sw_vers -productVersion)
```

Comment:
`$osMontereyVersion = [SemVer]$(sw_vers -productVersion)` -> `[SemVer]$osMontereyVersion = sw_vers -productVersion`

Reply:
Done. Thank you.